samples/drivers/led_is31fl3216a/boards/arduino_i2c.overlay 

```diff
@@ -0,0 +1,19 @@
+/*
```

Comment:
I think there is a trailing space at the end of the file name.

Reply:
Good catch :disappointed: